deno.lock

```diff
@@ -295,8 +295,7 @@
       }
     }
   },
-  "remote": {
-  },
+  "remote": {},
```

Comment:
Was this done by Deno? I think the next time the lockfile gets written it might undo this?

Reply:
Testing of another idea. My bad, sorry.